fli-iam/shanoir-ng
# shanoir-issue#2368-2: checkboxes + open multiple exams in ohif viewer

How to test:

Open a study with multiple exams
Check the exams you want to open:
![image](https://github.com/user-attachments/assets/5cb6e285-7716-4f01-852e-090bcd4d18c4)

Click the "Open in  viewer" button
In the left panel, you should see multiple studies and in the URL you should see the selected exam ids
![image](https://github.com/user-attachments/assets/755425c3-6086-4eb8-a5be-0b39a26b1399)
